indico/modules/rb/controllers/backend/common.py

```diff
@@ -23,6 +23,7 @@
     'end_dt': fields.DateTime(),
     'repeat_frequency': EnumField(RepeatFrequency),
     'repeat_interval': fields.Int(load_default=0),
+    'recurrence_weekdays': fields.List(fields.Str(), load_default=[]),
```

Comment:
You can use `validate=validate.OneOf([...])` here to filter out invalid values early